site/docs/foundations/size.mdx

```diff
@@ -57,13 +57,9 @@ The following example shows how components can use `--salt-size-base` and `--sal
 
 ## Icon
 
-`--salt-size-icon` controls the size of all icons throughout the design system. Icons are designed to be displayed no smaller than 12px x 12px; therefore this is set as a minimum measurement within the token's values. The result of this restriction can be seen in the Size tokens table. Following the ramp, HD should be 10px. However, the minimum size fixes it at 12px.
+`--salt-size-icon` controls the size of all icons throughout the design system. Icons are designed to be displayed no smaller than 12px x 12px; therefore this is set as a minimum measurement within the token's values. 
 
-The size of an icon can be changed within a density using the [Scale](./scale) Foundation. For HD, the scale is based on the expected value rather than the minimum. This means an HD icon starts at 12px. When scaled up (x2 its default size) it will be 20px rather than 24px, at x3 its default it will be 30px, etc.
-
-The Scale Foundation provides an example of `--salt-size-icon` scaling, including specific values for each density.
-
-You can find out more about icons and how to use them in the [Iconography](./iconography) Foundation.
+You can find out more about icons in the [Iconography](./iconography) Foundation.
```

Comment:
We should be consistent about whether links are: <link> foundation or the whole thing is the link. We also need to be consistent about capitalisation.

Reply:
Agreed, thanks for pointing it out - will edit this.